idseq/cli.py

```diff
@@ -87,6 +85,10 @@ def main():
         '--accept-all',
         action='store_true',
         help='Use this argument to automatically accept confirmation messages')
+    parser.add_argument(
```

Comment:
Why do we need to skip geosearch vs accepting all?
I am worried about having this as a parameter for regular use...

Reply:
I added the skip geosearch option to remove a potential point of failure in any upstream automation script. We have seen some failures of the location IQ API lately. That being said, if you feel it is premature, I would be happy to remove it from this PR. Keep in mind that location is still a required field in the metadata, regardless of geosearch.

Reply:
I know, but the whole project of geosearch is to make sure we have clean locations as much as possible. So, I would not encourage people to skip it during regular use. Feel free to consult with @cdebourcy and @oliviabholmes.
If strictly required for biohub use case, we could also do a fork with this option... 
What failures are you referring to?

Reply:
See https://jira.czi.team/browse/IDSEQ-2013 . 

As for encouraging people to skip it, our current default is to skip it in the confirmation prompt.
```
Is this correct (y/N)? y for yes or N to reject the match: 
```
Perhaps that is something we should change as well in this PR. 

In any case, I'm happy to remove the skip geosearch flag if anybody feels it's a bad idea.

Reply:
Yeah, i wonder why we chose the default as no. Humm, the no is also not a blocker for uploading?

But in this case also, skip is just for errors? Because accept-all would already accept this according to your change?

Reply:
> the no is also not a blocker for uploading?

Correct. And it is possible in the web UI to reject the location suggestions as well. 

>  in this case also, skip is just for errors

For errors, yes, and to give users full control over location text in automating the CLI.

Reply:
> give users full control over location text

This is what i would avoid, since i could see - if this feature was more used - people using it just for "laziness". 
Anyway, I see the arguments and my preference would be for still going through the location matching... but just check with product.

Reply:
I don't want to make it **easy** for people to opt out of using our location match.  Like Tiago said and I know that you are well aware Greg from the sample type project :) I want to try and encourage people to do things that keep our data clean. 

That being said, and with the issues we have seen with the Location API, I wouldn't want them to be blocked on upload if the location match returns incorrect results. They should be able to enter free text if they need to. 

My fear with adding the skip geosearch parameter is that I don't want to encourage people to take the "lazy" approach, like Tiago said. If there are specific use cases for the Biohub where you see this being a real blocker - let's talk about those. With the script you are writing to automate some of the upload process, would the user be required to confirm the location matches before continuing with the script?

Reply:
>  I don't want to encourage people to take the "lazy" approach

Keep in mind that currently the "lazy" approach would be to ignore the suggestions, the current default. If we want to encourage clean data, I suggest we change the current default.

The `skip` flag is something people would need to explicitly type out. 

That said, it seems like there is little support here for adding the feature, so I will take it out until we hear David wants.

> would the user be required to confirm the location matches before continuing with the script?

No, that is covered by `accept-all`.